metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -628,5 +639,33 @@ public void apply(final Metafix metafix, final Record record, final List<String>
     private static final String ERROR_STRING_OPTION = "error_string";
 
     private static final Random RANDOM = new Random();
+    private static String defaultValue;
```

Comment:
I'm sorry, but this is _bad_ ;) Not to mention, not thread-safe.

Reply:
ack. Fixed in ae0454ee1859a77b409aea081bcea55bd4f45564